scrapers/DoTheWife.yml

```diff
@@ -0,0 +1,36 @@
+name: Do The Wife
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - dothewife.com/update
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class="section-title"]/h4
+      Performers:
+        Name: //div[@class="model-rich"]/h4[@class="theme-color"]/a
+      Details:
+        selector: //p[@class="read-more"]/text()
+        postProcess:
+          - replace:
+              - regex: '^\s*:\s*'
+                with: ""
+      Tags:
+        Name:
+          selector: //div[@class="model-categories"]/a/text()
+      Studio:
+        Name:
+          fixed: Do The Wife
+      Image:
+        selector: //img[@class="video-banner"]/@src|//div[@class="vjs-poster"]/@style
+        postProcess:
+          - replace:
+              - regex: (?:.+)(\/usermedia\/.+\.jpg)(?:.+)
+                with: $1
+              - regex: "^/usermedia/"
+                with: "https://dothewife.com/usermedia/"
+driver:
+  useCDP: true
+  sleep: 4
```

Comment:
Is this even needed?
If it is for the image part, we can get it directly from the page source (`//video/poster`) eg the below seems to work without any cdp
```yaml
      Image:
        selector: //img[@class="video-banner"]/@src|//video/@poster
        postProcess:
          - replace:
              - regex: "^/usermedia/"
                with: "https://dothewife.com/usermedia/"
```

Reply:
Thank you for pointing this out. I have updated the PR with the above change.

PS: I had copied this from another scraper and adjusted it for dothewife.com. The CDP stuff were leftover from the original yml file.